stubs/docker/docker/context/context.pyi

```diff
@@ -1,47 +1,76 @@
-from _typeshed import Incomplete
+from collections.abc import Mapping
+from typing import Any, TypedDict, type_check_only
+
+@type_check_only
+class StorageData(TypedDict):
+    MetadataPath: str
+    TLSPath: str
+
+@type_check_only
+class Storage(TypedDict):
+    Storage: StorageData
+
+@type_check_only
+class Endpoint(TypedDict, total=False):
+    Host: str
+    SkipTLSVerify: bool
+
+@type_check_only
+class TLSMaterial:
+    TLSMaterial: Any
+
+@type_check_only
+class MetaMetaData(TypedDict, total=False):
+    StackOrchestrator: str
+
+@type_check_only
+class Metadata(TypedDict):
+    Name: str
+    Metadata: MetaMetaData
+    Endpoints: Mapping[str, Endpoint]
 
 class Context:
-    name: Incomplete
-    context_type: Incomplete
-    orchestrator: Incomplete
-    endpoints: Incomplete
-    tls_cfg: Incomplete
+    name: str
+    context_type: str | None
+    orchestrator: str | None
+    endpoints: Mapping[str, Endpoint]
+    tls_cfg: Mapping[str, Any]
```

Comment:
You could use `dict` here, as they seem to be managed by the class and that always uses `dict`, and is an input and output type. No strong opinion, though.